stable_baselines/a2c/a2c.py

```diff
@@ -249,7 +249,7 @@ def learn(self, total_timesteps, callback=None, seed=None, log_interval=100, tb_
                                                                       masks.reshape((self.n_envs, self.n_steps)),
                                                                       writer, self.num_timesteps)
 
-                self.num_timesteps += self.n_batch + 1
+                self.num_timesteps += self.n_batch
```

Comment:
thinking again about it, this line should be right after `runner.run()` no?
Otherwise, the first update will be `step=0` in tensorboard

Reply:
`_train_step` takes place before this line of update, so the first update will be `step=0` no matter what this line (:252) does. The example of logging the first step to be 0 can be seen at this [TensorFlow example](https://github.com/aymericdamien/TensorFlow-Examples/blob/master/examples/4_Utils/tensorboard_basic.py#L82).

Reply:
>takes place before this line of update

yes, that's what I'm proposing, moving the line before train_step call. Except if you think that's ok to keep it like that ;)

Reply:
I think it's just a matter of preference, but I would prefer the original version because it's easier to process the log as `for update_index in range(num_updates)` and it's consistent with the approaches in other algorithms. If we move the line, we should also consider moving `total_episode_reward_logger`, otherwise the timesteps logged in `episode_reward` would also be incorrectly offset.